src/plugins/isup.lua

```diff
@@ -0,0 +1,36 @@
+url = require("socket.url")
```

Comment:
Both of these should be `local` variables (don't want to pollute the global namespace unnecessarily).